tools/travis/run_server.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+
+cp config/example/* config/
+
+#Apply test DB config for SQL connectivity
+rm config/dbconfig.txt
+cp tools/travis/dbconfig.txt config
+
+# Now run the server and the unit tests
+DreamDaemon paradise.dmb -close -trusted -verbose
+
+# Check if the unit tests actually suceeded
+cat data/clean_run.lk
```

Comment:
If the tests didnt succeed, this will exit due to an invalid file, and thus fail travis